nicsdru_origins_theme.theme

```diff
@@ -273,13 +276,15 @@ function nicsdru_origins_theme_form_user_pass_alter(&$form, FormStateInterface $
  * Implements hook_preprocess_facets_item_list().
  */
 function nicsdru_origins_theme_preprocess_facets_item_list(array &$variables) {
```

Comment:
not sure that you need this proprocess function on the facet list. There is already a twig template that handles all the styling for facet links.

Reply:
This hook was already in the theme file, all I added was an additional if check. Does that mean we can remove this entirely?

Reply:
I looked at this again, think it might need to stay.